src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    double area();
```

Comment:
The method 'area()' should not be in the 'Drawable' interface. This method is related to calculating the area, which is a different responsibility from drawing. Consider creating a separate interface for area calculation, such as 'AreaCalculator', to adhere to the Single Responsibility Principle.